### PR TITLE
Explain the role invariance plays

### PR DESCRIPTION
**Pull request by burdges**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -226,6 +226,11 @@ impl<'a, T> RentToOwn<'a, T> {
     /// 20 | }
     ///    | - mutable borrow ends here
     /// ```
+    ///
+    /// Importantly, if `'a` is not `'b` then one cannot coerse from
+    /// `&'a mut RentToOwn<'a,Thing>` to `&'b mut RentToOwn<'b,Thing>`
```
Comment (fitzgen) [inline src/lib.rs]: Nitpick: needs a space after the commas, before `Thing`:

> `&'a mut RentToOwn<'a, Thing>` to `&'b mut RentToOwn<'b, Thing>`

**Review by fitzgen — COMMENTED** [review]
Thanks! Just one nitpick below and we can merge this :)

**Comment by burdges** [comment]
Added the requested spaces and rebased.

**Review by fitzgen — APPROVED** [review]
Thanks :)

**Pull request merged**